action.yaml

```diff
@@ -0,0 +1,59 @@
+# action.yml
+name: "codacy-analysis-cli"
+description: "Execute Codacy code analysis using your remote Codacy configuration"
+branding:
```

Comment:
Can we add any examples in this file? Or extra docs in the parameters?

Reply:
It's possible to add a description to each input parameter.

Reply:
I think I misunderstood you. The branding property determines the icon of your action in GH marketplace. We are using the check with a drak gray background, you can see how it looks here: https://haya14busa.github.io/github-action-brandings/ . 

If your account is verified, I think you can choose an image for your action.

Reply:
Not regarding branding, just regarding the action in general.

Reply:
The only other property missing was `author`